src/products/mozilla_org/testing.md

```diff
@@ -52,28 +52,62 @@ You can make linguistic changes directly in [Pontoon](https://pontoon.mozilla.or
+* An activated page falling far behind with multiple updates will create bad user experience. The page will risk being disabled but deactivation will be done manually.
+* In some cases, a page receives a major update that require a complete rewrite of the template. If this happens, the old template is kept online only for a defined period of time. When removed, it will cause the URL to redirect users to the English version if the intended localized version is not ready.
 
 ### Sync and update frequencies
 
+Updated translations are pushed to the production server regularly throughout the day.
+
 * Pontoon syncs every 20 minutes to the repository.
-* The Dev server updates every 15 minutes.
+* GitHub update is pushed to production every 5 minutes.
 
-If you work on Pontoon, it is safe to say that it will take less than an hour to see your changes reflected on the dev server.
+It is safe to say that it will take less than an hour to see your changes reflected on the production server.
 
 When a project has a firm deadline to meet, it will be communicated through the [dev-l10n-web mailing list](https://lists.mozilla.org/listinfo/dev-l10n-web). Be sure to sign up so you receive important community wide information on web related projects. You can also check out the deadline at page level in Pontoon.
 
+## Testing Fluent migration
+
+This section focuses on review and test after a page is migrated to Fluent.
+
+### Reviewing and updating [brands.ftl](https://github.com/mozilla-l10n/www-l10n/blob/master/en/brands.ftl)
+
+This new file captures all the brand and product names that appear in all the pages on mozilla.org as of now. As a general rule, brands and product names remain unchanged. The exceptions are:
+
+* Account/Accounts in Firefox Account/Accounts
+* Sync in Firefox Sync
+* Browser in Firefox Browser
+
+If your community decides to translate any of these products or feature names, translate it once and translate it in this file. Be sure to have a consensus among active contributors what the translation should be before updating this file. This practice will ensure term use consistency across all pages.  
+
+Please refer to the guidelines for brands and trademark when reviewing and updating this file.
+
+### Reviewing and updating migrated files
+
+The dashboard contains fewer files than before. This list will grow over time as the mozilla.org team continues migrating from .lang to Fluent. Keep track of what is reviewed and what is pending for a review. Here are a few things to check:
+* On the project dashboard, if there is orange color in the status progress bar, click on the file.  Check the strings with the orange color indicator first. If the localized string is absent of a variable, chances are, the product or brand name is translated, transliterated or declensed. Insert the variable to replace the translated/transliterated/declensed term.
```

Comment:
nit: you can click on the orange number of warnings.

> If the localized string is absent of a variable

is missing a term reference (not variable).